pkg/controller/evpa/effective_vpa_controller.go

```diff
@@ -172,30 +195,32 @@ func recordMetric(evpa *autoscalingapi.EffectiveVerticalPodAutoscaler, status *a
 
 		recommendCpu := container.Target[v1.ResourceCPU]
 		currentCpu := resourceRequirement.Requests[v1.ResourceCPU]
+		labels["resource"] = v1.ResourceCPU.String()
 		if currentCpu.Cmp(recommendCpu) > 0 {
 			// scale down
 			currCopy := currentCpu.DeepCopy()
 			currCopy.Sub(recommendCpu)
-			metrics.EVPACpuScaleDownMilliCores.With(labels).Set(float64(currCopy.MilliValue()))
+			metrics.EVPACpuScaleDownMilliCores.With(labels).Set(float64(currCopy.MilliValue()) / 1000.)
 		} else if currentCpu.Cmp(recommendCpu) < 0 {
 			// scale up
 			recommendCopy := recommendCpu.DeepCopy()
 			recommendCopy.Sub(currentCpu)
-			metrics.EVPACpuScaleUpMilliCores.With(labels).Set(float64(recommendCopy.MilliValue()))
+			metrics.EVPACpuScaleUpMilliCores.With(labels).Set(float64(recommendCopy.MilliValue()) / 1000.)
 		}
 
 		recommendMem := container.Target[v1.ResourceMemory]
 		currentMem := resourceRequirement.Requests[v1.ResourceMemory]
+		labels["resource"] = v1.ResourceMemory.String()
 		if currentMem.Cmp(recommendMem) > 0 {
 			// scale down
 			currCopy := currentMem.DeepCopy()
 			currCopy.Sub(recommendMem)
-			metrics.EVPAMemoryScaleDownMB.With(labels).Set(float64(currCopy.Value() / 1024 / 1024))
+			metrics.EVPAMemoryScaleDownMB.With(labels).Set(float64(currCopy.Value()))
```

Comment:
Use MB unit here.

Reply:
> Use MB unit here.

same reason as above, i think we should not use the unit here, keep the original bytes. and use label to denote the unit

Reply:
ok， please also change variable name.